README.md

```diff
@@ -165,3 +167,68 @@ definition location: /Volumes/src/orc/extras/struct0/src/a.cpp:3
 ```
 
 What line and file the object was declared in. So line 3 of `a.cpp` in this example.
+
+# The ORC Test App (`orc_test`)
```

Comment:
The description shouldn't include time, as it will be lost to...time. :) "a unit test application has been added to the system" should probably be "a unit test application is provided..." etc.

Reply:
Change to, "A unit test application is provided to ensure that ORC is catching what is purports to catch".